crates/toml/src/lib.rs

```diff
@@ -165,7 +165,7 @@ mod table;
 
 #[cfg(feature = "parse")]
 #[doc(inline)]
-pub use crate::de::{from_str, Deserializer};
+pub use crate::de::{from_reader, from_slice, from_str, from_value, Deserializer};
```

Comment:
Is there a reason the `from`s are being added but not the `to`s?

See also #349

Reply:
Indeed!

I added to_writer/to_writer_pretty.